src/systems/CMakeLists.txt

```diff
@@ -103,5 +103,6 @@ add_subdirectory(touch_plugin)
 add_subdirectory(triggered_publisher)
 add_subdirectory(user_commands)
 add_subdirectory(velocity_control)
+add_subdirectory(vehicle_damage)
```

Comment:
Nit: alphabetize

Reply:
Fixed in 7c84f8ed87f1f3997138d1859c9f95a3d63078e8